admin_manual/configuration_server/background_jobs_configuration.rst

```diff
@@ -60,6 +60,48 @@ You can verify if the cron job has been added and scheduled by executing::
 
 .. note:: On some systems it might be required to call **php-cli** instead of **php**.
 
-.. note:: Please refer to the crontab man page for the exact command syntax.
+Available Background Jobs
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A number of existing background jobs are available to be run just for specific tasks. 
```

Comment:
@settermjd sorry for not noticing this earlier. Currently, these jobs are only needed on large instances. They are also run as background jobs. But if the number of users increases > 1000-3000 or LDAP becomes a bottleneck admins can delete a few entries in the oc_jobs table and replace it with the corresponding occ command:

OCA\Files_Trashbin\BackgroundJob\ExpireTrash -> occ trashbin:expire
OCA\Files_Versions\BackgroundJob\ExpireVersions -> occ versions:expire
OCA\DAV\CardDAV\SyncJob -> occ dav:sync-system-addressbook
OCA\Federation\SyncJob -> occ federation:sync-addressbooks

Schedule them daily.